src/containers/my-resources/lessons-container/LessonsContainer.styles.ts

```diff
@@ -6,6 +6,18 @@ const captionTitle = {
 }
 
 export const styles = {
+  table: {
+    '& td,th': {
+      '&:first-of-type': {
+        borderTopLeftRadius: '10px',
+        borderBottomLeftRadius: '10px'
+      },
+      '&:last-of-type': {
+        borderTopRightRadius: '10px',
+        borderBottomRightRadius: '10px'
+      }
+    }
+  },
```

Comment:
duplicated style, If we are using same styles for myCooperation page tables, make sense to move it out or import in another styles

Reply:
Fixed